R/agglomerate.R

```diff
@@ -49,17 +51,19 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting 
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting 
```

Comment:
This is not actually removing empty ranks. This is for removing empty columns from rowData (column can be anything, also those that are not ranks). The parameter name is ok, but this behavior should be changed. (only rank colum should be removed). That modification must be done on other PR.

Reply:
https://github.com/microbiome/mia/issues/568